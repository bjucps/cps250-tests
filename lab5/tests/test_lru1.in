ape
bat
cat
ape
bat
cat
dog
bat
emu
dog
cat
