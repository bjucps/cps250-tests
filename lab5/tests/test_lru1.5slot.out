ape: MISS (1/5 full; no eviction)
bat: MISS (2/5 full; no eviction)
cat: MISS (3/5 full; no eviction)
ape: HIT
bat: HIT
cat: HIT
dog: MISS (4/5 full; no eviction)
bat: HIT
emu: MISS (5/5 full; no eviction)
dog: HIT
cat: HIT
--------------------------------
FINAL CACHE CONFIGURATION:
--------------------------------
cat
dog
emu
bat
ape
